cmd/core/pdr_creation_context.go

```diff
@@ -78,6 +79,13 @@ func (pcc *PDRCreationContext) extractPDR(pdr *ie.IE, spdrInfo *SPDRInfo) error
 		}
 		return fmt.Errorf("F-TEID IE is missing")
 	} else if ueIP, err := pdr.UEIPAddress(); err == nil {
+		if ueIP.Flags&(1<<2) != 0 {
```

Comment:
It's better to extract a method from here. We chech flags due to no corresponding method provided in pfcp library. So just implement such method here